src/engine/specs/CoreSpecs.ts

```diff
@@ -90,6 +91,15 @@ export interface IWallSpec extends IBaseSimObjectSpec {
   height?: number;
 }
 
+/**
+ * Spec for a Perimeter
+ */
+export interface IPerimeterSpec {
+  type: "perimeter";
```

Comment:
Don't need this field. Since a `IPerimeterSpec` is a standalone interface, and it goes into the `perimeter` field, it should be self-explanatory enough.

(side note: the `type` field is used in interfaces extending `IBaseSimObjectSpec` to help identify the interfaces and help the typescript autocomplete narrow down the fields accordingly)